components/CommonCompGroupUserList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { ScrollView, StyleSheet, View } from 'react-native';
+import CommonCompGroupUser from './CommonCompGroupUser';
+
+
+const CommonCompGroupUserList = ({ groupMembers }) => {
+
+    return (
+        <View>
+            <ScrollView
+                horizontal={true}
+                //contentContainerStyle={{ width: `${groupMembers.length*30}%` }}
+                showsHorizontalScrollIndicator={false}
+                scrollEventThrottle={200}
+                decelerationRate="normal">
```

Comment:
Just a sanity check, does this allow horizontal scroll in the mobile version?

Reply:
Yep, at least it does on Android